examples/.github/workflows/security-hub-jira-sync.yml

```diff
@@ -18,36 +18,25 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
 
-      - name: Configure direnv
-        uses: HatsuneMiku3939/direnv-action@v1
-
       - name: Install Node
         uses: actions/setup-node@v3
         with:
-          node-version-file: .nvmrc
-
-      - name: Node cache
-        uses: actions/cache@v2
-        with:
-          path: "**/node_modules"
-          key: ${{ github.workflow }}-${{ github.job }}-${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+          node-version: 16
 
+      # AWS credentials must be provided to get findings from security hub; how you inject credentials can vary.
       - name: Configure AWS credentials
         uses: aws-actions/configure-aws-credentials@v1-node16
         with:
           role-to-assume: ${{ secrets.AWS_OIDC_ROLE_TO_ASSUME }}
           aws-region: us-east-1
           role-duration-seconds: 10800
 
+      - run: npm install
+
       - name: Invoke Security Hub Jira Sync
-        id: runningStages
         env:
           JIRA_HOST: qmacbis.atlassian.net
-          JIRA_USERNAME: xxx
-          JIRA_TOKEN: xxx
           JIRA_PROJECT: OY2
-          JIRA_CLOSED_STATUSES: "Done"
-          PROJECT: cms-bigmac
-        run: |
-          npm i -g ts-node
-          ts-node src/run-sync.ts
+          JIRA_USERNAME: ${{ secrets.JIRA_USERNAME }}
+          JIRA_TOKEN: ${{ secrets.JIRA_TOKEN }}
+        run: node run.js
```

Comment:
@mdial89f will this run.js file exist?  I don't believe the run.js will be built at this point.